src/System.Runtime.Intrinsics.Experimental/src/Configurations.props

```diff
@@ -0,0 +1,10 @@
+﻿<Project DefaultTargets="Build">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netcoreapp-Windows_NT;
```

Comment:
it doesn't seem like you need the OS specific configurations, because you're not using any of the `TargetsWindows` or `TargetsLinux` properties in the csproj.

Reply:
So this will generate runtime specific assets in the package, but it seems like your implementation is not OS specific.

Reply:
A lot of this was just copied from the existing `System.Runtime.Intrinsics` project (and validated to be the same/similar as the previous `S.R.I.Experimental` project).

If we can remove it here, we could probably also remove it there; correct?

Reply:
From looking at the csproj yes. I'm not sure if we do runtime specific stuff when referencing corelib from runtime though, but I don't think we do. @ericstj to confirm

Reply:
In the past this was necessary for the corelib reference. I’m out of the loop on the current build infra on corefx, so if we changed it in the past year to fixed that, then we can change this.

Reply:
Talked with @ericstj in person and he indicated this is still needed right now.

Reply:
Yep: since corelib doesn’t provide a reference assembly and exposes different types per runtime we play it safe by ensuring runtime referencing libs cross compile (and run api compat per config).